src/droplet.F

```diff
@@ -126,9 +133,20 @@ subroutine droplet_driver(dt,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,    &
       integer :: tmp_flag
 #endif
 
+      double precision :: t_s,t_f,t_findloc,t_interp,t_BEint,t_feed,t_s1,t_f1,t_BEprep,t_radsolver,t_gauss,t_LV,t_neighbor
+
+
+      t_s = mpi_wtime()
+
+      !!! DHR: From what I can tell this is useless -- it only resets seed for the intrinsic random_number routine
```

Comment:
I agree and I thought you would remove these calls and take the `randomNumbers` from the `init3d` subroutine to pull new random numbers.

Reply:
Okay I have removed this